client/modules/Booking/Booking.js

```diff
@@ -117,19 +120,45 @@ labels = {
 		this.setState({ endDate: date });
 	}
 
+	handleBooking(evt){
+		if (this.isFormInvalid()) {
+			evt.preventDefault();
+			return;
+		}
+		else{
+			evt.preventDefault();
+			http.client().post('/booking/', {
+				userid: this.state.userid,
+				car: this.state.carid,
+				startAt: this.state.startDate,
+				endAt: this.state.endDate,
+			})
+			.then(res => {
+				console.log(res);
+				this.setState({booked: true});
+			})
+			.catch(err => console.log(err));
+		}
+		//alert(`Signed up with email: ${email} password: ${password1} mobile: ${mobile} license: ${license}`);
+}
+
 	handleSubmit(evt){
        if (this.isFormInvalid()) {
          evt.preventDefault();
```

Comment:
can we move evt.preventDefault to the first line of this method? currently we are calling it as the first step of both the if and the else statements

Reply:
Gimme a sec, will address as soon as I get my mac running